pkg/ddevapp/global_dotddev_assets/commands/db/mysqldump.example

```diff
@@ -1,8 +1,16 @@
 #!/usr/bin/env bash
 
 ## #ddev-generated
-## Description: run mysqldump in web container
+## Description: run mysqldump or mariadb-dump in web container
```

Comment:
This is fine. I do wonder if this example (or the other examples) actually serve a purpose more than the docs.